packages/amplify-category-api/src/provider-utils/awscloudformation/service-walkthroughs/appSync-rds-walkthrough.ts

```diff
@@ -70,6 +88,18 @@ export async function serviceWalkthrough(context: $TSContext, datasourceMetadata
   };
 }
 
+async function getClusterResourceIdFromArn(arn: string|undefined, AWS) {
+  if (!arn) {
+    return;
```

Comment:
Is there a case when the arn is empty? if so, could you pl add a comment here. If not do we need to throw an error ?